src/solve.jl

```diff
@@ -147,32 +182,30 @@ end
 solve(system::Union{ContinuousSystem, DiscreteSystem}, options::Pair{Symbol,<:Any}...) = solve(system, Options(Dict{Symbol,Any}(options)))
 
 """
-    project(Rsets, n, options, [transformation_matrix])
+    project(Rsets, options)
 
 Projects a sequence of sets according to the settings defined in the options.
 
-INPUT:
+### Input
 
-- ``Rsets``                 -- sequence of sets
-- ``n``                     -- dimension
-- ``options``               -- options structure
-- ``transformation_matrix`` -- (optional, default: `nothing`) transformation matrix
+- `Rsets`   -- solution of a reachability problem
+- `options` -- options structure
 """
 function project(Rsets::Union{Vector{CartesianProductArray}, Vector{HPolygon}},
-                 n::Int64, options::Options, transformation_matrix=nothing)
-
-    RsetsProj = project_reach(
-        options[:plot_vars],
-        n,
-        options[:δ],
-        Rsets,
-        options[:algorithm],
-        ɛ=options[:ɛ],
-        transformation_matrix=transformation_matrix,
-        projection_matrix=options[:projection_matrix]
-        )
+                 options::Options; transformation_matrix=nothing)
```

Comment:
Why is `transformation_matrix` excluded from the doc and still kept as an argument?

Reply:
i forgot, thanks for pointing this out

Reply:
I still see two "RsetsSolution"s ;)

Reply:
line num?

Reply:
fixed

Reply:
`plot_recipes.jl`, l. 2 and 17